### PR TITLE
[16.0][MIG] purchase_stock_picking_invoice_link: Migration to 16.0

### PR DESCRIPTION
**Pull request by JavierIniesta**
Issue: #1101

### CODE REVIEW

**Comment by JavierIniesta** [comment]
We have done functional tests and it works correctly.

**Review by rousseldenis — APPROVED** [review]
Code review

File: purchase_stock_picking_invoice_link/tests/test_purchase_stock_picking_invoice_link.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
+# Copyright 2019 Vicent Cubells <pedro.baeza@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import fields
+from odoo.tests import Form, common, tagged
+
+
+@tagged("-at_install", "post_install")
```
Comment (lmignon) [inline purchase_stock_picking_invoice_link/tests/test_purchase_stock_picking_invoice_link.py:7]: useless...  but I don't want to reopen an endless debate

**Review by lmignon — APPROVED** [review]
Code review

File: purchase_stock_picking_invoice_link/tests/test_purchase_stock_picking_invoice_link.py (hunk 1 above)
Comment (pedrobaeza) [reply purchase_stock_picking_invoice_link/tests/test_purchase_stock_picking_invoice_link.py:7]: Not this time, Laurent. For having a proper CoA, this should be put on post install. It's a requirement put on everything depending on `account`. Please see as something related (but only partial history) https://github.com/odoo/odoo/commit/f34d8632572dfe8911b62471882eae60f6e6ae72

Comment (lmignon) [reply purchase_stock_picking_invoice_link/tests/test_purchase_stock_picking_invoice_link.py:7]: Thank you @pedrobaeza for the explanation. :+1:

**Pull request merged**